patch.yml

```diff
@@ -13,6 +13,54 @@
 # limitations under the License.
 
 ---
+- name: Play | Set correct RDBMS release and set fact if this is a single node inventory
```

Comment:
How do we ensure that this code only runs when there's a single node?

Reply:
The following snippet ensures that it be run irrespective of the number of cluster nodes:
```
    - name: set fact is this is a single node inventory - needed for RAC patching (MOS note 2336186.1)
      set_fact:
        non_rolling_clause: "{{ ' -nonrolling' if groups['dbasm'] | length==1 else ' '}}"
```